data/Gulam-Mustafa231999.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Gulam Mustafa",
+  "bio": "Techie Teacher at Techie Tessie.",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/Gulam-Mustafa231999",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/GulamMu79035761",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/gulam-mustafa-706933257",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on Facebook",
+      "url": "https://www.facebook.com/gulam.mustafa.3781",
+      "icon": "FaFacebook"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://instagram.com/gulammustafa954",
+      "icon": "FaInstagram"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started learning javaScript",
+      "date": "July 2023",
+      "icon": "FaJavaScript",
```

Comment:
This is not a valid icon, consider viewing the available icons [here](https://linkfree.io/docs/quickstart-json)
![js](https://github.com/EddieHubCommunity/LinkFree/assets/19821445/9ed0069f-df09-457c-b186-672112d7be39)
```suggestion
```

Reply:
Good spot - but as it has been several days and this is not a blocker, I will merge this Pull Request. 

@Gulam-Mustafa231999 if you wish to change the icon or make any other changes to your Profile, then you can make edits either through JSON or through our Forms.  At that stage I suggest you take a look at the Editing Guides in Docs: https://linkfree.io/docs